db/migrate/20230316102754_create_comments.rb

```diff
@@ -0,0 +1,11 @@
+class CreateComments < ActiveRecord::Migration[7.0]
+  def change
+    create_table :comments do |t|
+      t.text :Text
+      t.references :user, null: false, foreign_key: true
+      t.references :post, null: false, foreign_key: true
+
```

Comment:
- [ ] Please make sure to follow the given diagram, as the reference to the users table should be called `author` instead of `user`.